lib/manager/npm/extract/pnpm.spec.ts

```diff
@@ -0,0 +1,150 @@
+import _yaml from 'js-yaml';
+import { getFixturePath, getName } from '../../../../test/util';
+import { setAdminConfig } from '../../../config/admin';
+import { logger } from '../../../logger';
+import * as _fs from '../../../util/fs';
```

Comment:
```suggestion
import * as fs from '../../../util/fs';
```
No rename if not necessary.

Reply:
changed by 705fa8391f6e08cb2d951e2bbf1234d7b396b7df